app/src/main/res/values/styles.xml

```diff
@@ -341,17 +341,16 @@
 
     <style name="SimpleDraweeViewPlaceholder">
         <item name="actualImageScaleType">centerCrop</item>
-        <item name="placeholderImage">?attr/thumbnail_image_placeholder</item>
         <item name="placeholderImageScaleType">fitCenter</item>
```

Comment:
Since the `placeholderImage` is removed, so set up `placeholderImageScaleType` is unnecessary.